src/Lifecycle.ts

```diff
@@ -66,6 +66,7 @@ import { OverwriteLoginPayload } from "./dispatcher/payloads/OverwriteLoginPaylo
 import { SdkContextClass } from "./contexts/SDKContext";
 import { messageForLoginError } from "./utils/ErrorUtils";
 import { completeOidcLogin } from "./utils/oidc/authorize";
+import { getErrorMessage } from "./utils/oidc/error";
```

Comment:
I feel like having a function called `getErrorMessage` imported into this file is pretty confusing, when that function only deals with OIDC errors.

Possibly the function should be called `getOidcErrorMessage`. Or we could do something like:

```suggestion
import * as OidcErrorUtils from "./utils/oidc/error";
```

... and then call `OidErrorUtils.getErrorMessage` below.